dts/arm/st/u5/stm32u595.dtsi

```diff
@@ -115,7 +115,9 @@
 
 	otghs_phy: otghs_phy {
 		/* Clock source defined by USBPHYC_SEL in */
```

Comment:
Comment should be updated:
```suggestion
		/* Clock source defined by USBPHYC_SEL() in usbotg_hs `clocks` */
```

Also, I think the following section should be updated in this file to match the comment (`ICKLK_SEL` -> `USBPHYC_SEL`?):
https://github.com/zephyrproject-rtos/zephyr/blob/dfbcea777a10e27d12539a4fd85b5203f5421151/dts/arm/st/u5/stm32u595.dtsi#L109-L110

Reply:
Fixed!